define({
    errorMessage: 'Leider ist bei der Ausführung des Auftrags \'${0}\': ${1} ${2} ein Fehler aufgetreten.',
    jobCompletedSuccessfully: 'Der Auftag wurde erfolgreich abgeschlossen.',
    unexpectedErrorMessage: 'Bei der Ausführung des Auftrags \'${0}\' ist leider ein unerwarteter Fehler aufgetreten.',
    repeatIndefinitely: 'Unbegrenzt wiederholen',
    cancelButtonCaption: 'Abbrechen',
    defaultProgressDialogTitle: 'Ausgeführter Auftrag',
    closeButtonCaption: 'Schließen'
});